services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/TransactWriteItemsEnhancedRequest.java

```diff
@@ -52,8 +52,25 @@
 
     private final List<TransactWriteItem> transactWriteItems;
 
+    private String clientRequestToken;
+
     private TransactWriteItemsEnhancedRequest(Builder builder) {
         this.transactWriteItems = getItemsFromSupplier(builder.itemSupplierList);
+        this.clientRequestToken = builder.clientRequestToken;
+    }
+
+    /**
+     * Sets the ClientRequestToken as a specific one
+     */
+    public void setClientRequestToken(String clientRequestToken) {
+        this.clientRequestToken = clientRequestToken;
+    }
+
+    /**
+     * Gets the ClientRequestToken of this enhanced request.
+     */
+    public String getClientRequestToken() {
+        return clientRequestToken;
     }
```

Comment:
We don't usually prefix getters with 'get'. I also think this is out of order and should go below the static methods.